include/procmapsarea.h

```diff
@@ -53,6 +53,11 @@
 #define DELETED_FILE_SUFFIX " (deleted)"
 
 #define FILENAMESIZE        1024
+#define ONEMB (uint64_t)(1024 * 1024)
+#define ONEGB (uint64_t)(1024 * 1024 * 1024)
```

Comment:
I don't think we need `uint64_t` here.

Reply:
if this is removed, it causes an assertion error in `mmap.c/getNextAddr`.